### PR TITLE
New sidebar colors

### PR DESCRIPTION
**Pull request by sampoonachot**
Non-Genuine MongoDB Modal
* Added 3px radius to banner message
* Added a period before "learn more" and moved the btn-link to the same line
Screen Shot 2019-07-29 at 11 27 04 AM

Warning Pill
* Made the border color more opaque
* Reduced font size to 11px
* Left aligned the pill

Sidebar 
* Swapped the old grays with the new gray palette
* In the instance stats, swapped the label colors and text sizes
* Shifted the copy to the right by 36px
Screen Shot 2019-07-29 at 11 27 22 AM
Screen Shot 2019-07-29 at 11 27 24 AM
Screen Shot 2019-07-29 at 11 27 30 AM

### CODE REVIEW

File: src/components/sidebar-database/sidebar-database.less [hunk 1]
```diff
@@ -81,7 +81,8 @@
 
     &-is-active,
     &-is-active:hover {
-      background-color: @compass-sidebar-active-background-color;
+      background: @compass-sidebar-base-background-color;
+      @compass-sidebar-base-background-color: lighten(hsl(201, 11%, 41%), 20%); // item-is-active
```
Comment (sampoonachot) [inline src/components/sidebar-database/sidebar-database.less]: @fredtruman for some reason I'm not able to see the active state, but this is the color that it's supposed to adopt.

**Comment by sampoonachot** [comment]
@durran I thought that the connection details and header lived in `compass-instance-header` but you mentioned that it's not part of 1.19.0? if you can point me to the right file I can help modify it! thank you

File: src/components/sidebar-database/sidebar-database.less (hunk 1 above)
Comment (sampoonachot) [reply src/components/sidebar-database/sidebar-database.less]: @durran think i might need that piece of code to view the instance header after all

File: src/components/sidebar-instance-stats/sidebar-instance-stats.less [hunk 2]
```diff
@@ -57,8 +57,9 @@
     transition: all 150ms ease;
 
     &:hover {
-      background: #42494f;
-      color: #fff;
+      background: @compass-sidebar-hover-background-color;
```
Comment (imlucas) [inline src/components/sidebar-instance-stats/sidebar-instance-stats.less]: This looks a bit funny. Will chat through it on Slack and come back.

File: src/assets/less/compass/variables/_palette.less [hunk 3]
```diff
@@ -3,27 +3,27 @@
...
-@gray0: #313030;
-@gray1: #494747;
-@gray2: #615f5f;
-@gray3: #807f7f;
-@gray4: #a09f9e;
-@gray5: #bfbfbe;
+@gray0: #061621; // black
+@gray1: #21313C; // gray dark3
+@gray2: #3D4F58; // gray dark2
+@gray3: #3D4F58; // TODO: deprecate
```
[the remark this answers is not in the document]
Comment (sampoonachot) [reply src/assets/less/compass/variables/_palette.less]: Does it matter if `@gray3` `@gray9` and `@gray10` are used elsewhere in the app? If not, happy to remove them if they only matter to this plugin.

**Comment by imlucas** [comment]
Related to: 

- mongodb-js/compass-deployment-awareness#9
- mongodb-js/compass-server-version#4
- mongodb-js/compass-ssh-tunnel-status#1

File: src/assets/less/compass/variables/_palette.less (hunk 3 above)
[the remark this answers is not in the document]
Comment (imlucas) [reply src/assets/less/compass/variables/_palette.less]: Removing them in this plugin won't impact if used elsewhere in the app or by another plugin

File: src/components/sidebar-collection/sidebar-collection.less [hunk 4]
```diff
@@ -3,7 +3,9 @@
 @compass-sidebar-default-text-color: @gray5;
 @compass-sidebar-base-background-color: @gray2;
 @compass-sidebar-hover-background-color: @gray1;
+@compass-sidebar-hover-background-color: lighten(hsl(204, 29%, 18%), 5%);
```
Comment (sampoonachot) [inline src/components/sidebar-collection/sidebar-collection.less]: @imlucas it looks like this works! lmk if you have any other suggestions

Comment (imlucas) [reply src/components/sidebar-collection/sidebar-collection.less]: yay! :shipit:

**Pull request merged**